Sources/UInt128.swift

```diff
@@ -429,7 +429,7 @@ extension UInt128 : BinaryInteger {
     // MARK: Instance Methods
 
     public var words: [UInt] {
-        guard self != UInt128.min else {
+        guard self != .min else {
```

Comment:
Changing this call to min here is inconsistent with 2 other places where it is called, plus it begs the question whether other properties/methods should be called in a similar way. I think I’d like to leave this as it was, at least for now.